lambdas/video/get_presigned_url.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import boto3
+from botocore.exceptions import NoCredentialsError
+
+from config import Config
+from utils.response import generate_error_response, generate_success_response
+from utils.request import parse_header
+
+# Logger
+logger = logging.getLogger('get presigned url')
+logger.setLevel(logging.INFO)
+
+formatter = logging.Formatter('[%(levelname)s]:%(funcName)s:%(lineno)d:%(message)s')
+
+if not logger.handlers:
+    ch = logging.StreamHandler()
+    ch.setFormatter(formatter)
+    logger.addHandler(ch)
+
+logger.propagate = False
+
+s3_client = boto3.client('s3')
+
+
+def handler(event, context):
+    """
+    Handles the lambda function call.
+    """
+    logger.info('Generating presigned url...')
+    user_id = 1
+    unique_id = generate_hash()
+    origin = parse_header(event)
+    try:
+        presigned_url = s3_client.generate_presigned_post(
+            Bucket=f'{Config.ENVIRONMENT}-data-question-response-video',
+            Key=f'{user_id}_{unique_id}.webm',
```

Comment:
why do you use this key name? right now we are using '{interview_id}/{question_id}.webm' so we can fetch the video by the question data